griptape/mixins/media_artifact_file_output_mixin.py

```diff
@@ -7,11 +7,11 @@
 from typing import Optional
 
 if TYPE_CHECKING:
-    from griptape.artifacts import ImageArtifact
+    from griptape.artifacts import MediaArtifact
 
 
 @define(slots=False)
-class ImageArtifactFileOutputMixin:
+class MediaArtifactFileOutputMixin:
```

Comment:
I think we'll want to accept some sort of artifact here because we fall back to the artifact name as output filename if one isn't provided (if `output_dir` is set and we might expect multiple artifacts to end up there). Agreed that there's no reason to limit ourselves to `MediaArtifacts`, though.